### PR TITLE
Rad dino model onboarding

### PR DESCRIPTION
**Pull request by roopavidhya**
Rad dino model onboarding

### CODE REVIEW

**Comment by roopavidhya** [comment]
image

File: assets/models/system/microsoft-rad-dino/description.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+}
+```
+
+**APA:**
+
+> Pérez-García, F., Sharma, H., Bond-Taylor, S., Bouzid, K., Salvatelli, V., Ilse, M., Bannur, S., Castro, D.C., Schwaighofer, A., Lungren, M.P., Wetscherek, M.T., Codella, N., Hyland, S.L., Alvarez-Valle, J., & Oktay, O. (2024). *RAD-DINO: Exploring Scalable Medical Image Encoders Beyond Text Supervision*. ArXiv, abs/2401.10815.
+
+## Model card contact
+
+Fernando Pérez-García ([`fperezgarcia@microsoft.com`](mailto:fperezgarcia@microsoft.com)).
```
Comment (naiteeks) [inline assets/models/system/microsoft-rad-dino/description.md:240]: We can remove this section. We generally do not put the contact information of one person in the model card

File: assets/models/system/microsoft-rad-dino/model.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: huggingface/microsoft-rad-dino/1733475346/mlflow_model_folder
```
Comment (naiteeks) [inline assets/models/system/microsoft-rad-dino/model.yaml:3]: Is the container path correct?

File: assets/models/system/microsoft-rad-dino/description.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+<!--fp32, fp16 mixed precision, bf16 mixed precision, bf16 non-mixed precision, fp16 non-mixed precision, fp8 mixed precision -->
+
+## Evaluation
+
+<!-- This section describes the evaluation protocols and provides the results. -->
+
+Our evaluation is best described in the [manuscript](https://arxiv.org/abs/2401.10815).
+
+<!-- ### Testing data, factors & metrics
```
Comment (naiteeks) [inline assets/models/system/microsoft-rad-dino/description.md:167]: This section is just a bunch of comments. Suggestions to remove it, else will look incomplete and odd

File: assets/models/system/microsoft-rad-dino/model.yaml (hunk 2 above)
Comment (roopavidhya) [reply assets/models/system/microsoft-rad-dino/model.yaml:3]: Yes, this is autogenerated by the pipeline

**Pull request merged**